cockroachcloud/use-managed-service-backups.md

```diff
@@ -25,7 +29,7 @@ Cockroach Labs runs [full cluster backups](../{{site.current_cloud_version}}/tak
 {{site.data.alerts.callout_info}}
 Currently, you can only restore [databases](#restore-a-database) and [tables](#restore-a-table) to the same cluster that the backup was taken from.
 
-In the meantime, you can [back up and restore data manually](run-bulk-operations.html) or [back up from a self-hosted CockroachDB cluster and restore into a {{ site.data.products.db }} cluster](#back-up-a-self-hosted-cockroachdb-cluster-and-restore-into-a-cockroachdb-cloud-cluster). Note that you cannot restore a backup of a multi-region database into a single-region database.
+In the meantime, you can [back up and restore data manually](take-and-restore-customer-owned-backups.html) or [back up from a self-hosted CockroachDB cluster and restore into a {{ site.data.products.db }} cluster](#back-up-a-self-hosted-cockroachdb-cluster-and-restore-into-a-cockroachdb-cloud-cluster). Note that you cannot restore a backup of a multi-region database into a single-region database.
 {{site.data.alerts.end}}
 
```

Comment:
In the dedicated section, perhaps change the sentence to include incremental backups as well: 

"A list of your full cluster backups displays on your cluster's **Backups** page." 

This is a remnant from when full backups were only available.